doc/Makefile

```diff
@@ -104,16 +101,18 @@ htmlhelp:
 	@echo "Build finished; now you can run HTML Help Workshop with the" \
 	      ".hhp project file in $(BUILDDIR)/htmlhelp."
 
-latex:
+latex: logo
```

Comment:
On a fresh context with no prior `make html`,  the `make latex` without logo requirement produces a `.tex` file which causes `xelatex` to crash.

Reply:
As long as we're fixing this, it looks like it's configured incorrectly. This will make it run the first time, but if `make logo` fails it doesn't recognize that and won't run it again. For me it failed with

```
ERROR:root:Return code is not 0: Command: convert /Users/aaronmeurer/Documents/Python/sympy/sympy/doc/./_build/logo/sympy-16px.png /Users/aaronmeurer/Documents/Python/sympy/sympy/doc/./_build/logo/sympy-32px.png /Users/aaronmeurer/Documents/Python/sympy/sympy/doc/./_build/logo/sympy-48px.png /Users/aaronmeurer/Documents/Python/sympy/sympy/doc/./_build/logo/sympy-64px.png /Users/aaronmeurer/Documents/Python/sympy/sympy/doc/./_build/logo/sympy-favicon.ico
```

(my `convert` command is broken on my machine)

Reply:
ah I did not notice because fortunately for me`make logo` works fine.  My `convert` is from the Fink package manager for macOS and is a bit old ``Version: ImageMagick 7.0.11-13 Q16 x86_64 2021-05-17 https://imagemagick.org``, but I update fink packages once a year at most. I am lucky my old system is still supported but I may reach end of of my luck next time.

Should I keep the `logo` here in view of your remark? I am not sure what this implies for this. Most people will build HTML in priority, and once done, the images are in place for `make latex` to pick them, (if `make logo` succeeded).

Reply:
Yes, it should be here. I was just pointing out that the `logo` target itself is not properly depending on its outputs.